spec/Reorder/ReordererSpec.php

```diff
@@ -132,41 +131,110 @@ function it_notifies_when_promotion_is_no_longer_available(
         $firstPromotion->getName()->willReturn('test_promotion_01');
         $secondPromotion->getName()->willReturn('test_promotion_02');
 
-        $reorder->getTotal()->willReturn(150);
         $reorder->getPromotions()->willReturn(new ArrayCollection());
 
         $moneyFormatter->format(100, 'USD')->willReturn('$1.00');
 
-        $reorderEligibilityChecker->check($order, $reorder)->willReturn([
-            [
-                'type' => 'info',
-                'message' => 'sylius.reorder.promotion_not_enabled',
-                'parameters' => [
-                    '%promotions%' => 'test_promotion_01, test_promotion02'
-                ]
-            ],
-            [
-                'type' => 'info',
-                'message' => 'sylius.reorder.previous_order_total',
-                'parameters' => ['%order_total%' => '$1.00']
-            ]
+        $reorderEligibilityCheckerResponse->getMessage()->willReturn(
+            EligibilityCheckerFailureResponses::REORDER_PROMOTIONS_CHANGED
+        );
+        $reorderEligibilityCheckerResponse->getParameters()->willReturn([
+            '%promotion_names%' => 'test_promotion_01, test_promotion_02'
+        ]);
+
+        $reorderEligibilityChecker->check($order, $reorder)->willReturn([$reorderEligibilityCheckerResponse]);
+
+        $orderFactory->createFromExistingOrder($order, $channel)->willReturn($reorder);
+        $entityManager->persist($reorder)->shouldBeCalled();
+        $entityManager->flush()->shouldBeCalled();
+
+        $this->reorder($order, $channel);
+    }
+
+    function it_checks_if_price_of_any_item_has_changed(
```

Comment:
Try to keep the order of parameters from `let` method here and in other method in spec files